### ISSUE TITLE
Add documentation about running notebooks in Google Colab.

### ISSUE THREAD

**Issue by ileasile** [Kotlin/kotlin-jupyter#443]
Now it's not clear (and now tested well yet) how to run Kotlin Notebooks in Google Colab. Document it. Use this notebook as a reference:
https://github.com/Kotlin/kotlin-jupyter/issues/440#issuecomment-1826705001

**Comment by ileasile** [comment]
@maxandersen Some users in our public Slack complain there is a problem connecting to the runtime (it's connecting infinitely)
![image-37.png](https://github.com/Kotlin/kotlin-jupyter/assets/20319593/5210f39c-d783-4c5d-b06f-93a850bbe1bf)

Don't you know what could be a reason and how to diagnose it?

**Comment by maxandersen** [comment]
works for me but let me re-test on fresh notebook.

To debug use Runtime > View runtime logs and look for suspicious log entries.

**Comment by maxandersen** [comment]
Can confirm that it is not working at the moment - not sure what goes wrong as the runtime log shows the runtime being executed. Something is broken ;/

**Comment by maxandersen** [comment]
other java jupyter kernels are working for me. weird.

**Comment by michalharakal** [comment]
Following the  comments, I wasn't able to stat Kotlin Kernel on Colab, still getting the connection issues. 
Any idea to get it running? Folks at deepjavalibrary have an shell script for bootstraping java kernel https://raw.githubusercontent.com/deepjavalibrary/d2l-java/master/tools/colab_build.sh ...

**Comment by maxandersen** [comment]
@michalharakal yes, what deepjavalibrary does is what I've implemented (just without requiring multiple steps) for java kernels - and works.

For some reason that "trick" does not work for kotlin kernel anymore despite they should be very similar.
